spec.emu

```diff
@@ -48,7 +48,9 @@ copyright: false
       1. Let _buffer_ be a new empty List.
       1. Repeat,
         1. Let _value_ be ? IteratorStepValue(_iterated_).
-        1. If _value_ is ~done~, return ReturnCompletion(*undefined*).
+        1. If _value_ is ~done~, then
+          1. If the number of elements in _buffer_ &lt; ℝ(_windowSize_), perform ? Yield(CreateArrayFromList(_buffer_)).
```

Comment:
```suggestion
          1. If _buffer_ is not empty, perform ? Yield(CreateArrayFromList(_buffer_)).
```

By construction it can never get as large as `_windowSize_`, and also we don't want to yield an empty array in the case that the input was actually empty.